web/public/js/init-master.js

```diff
@@ -102,11 +102,31 @@
     hiddenFields: function(cdUsersService){
       return cdUsersService.getHiddenFieldsPromise().then(winCb, failCb);
     },
-    championsForUser: function ($stateParams, cdUsersService) {
-      return cdUsersService.loadChampionsForUserPromise($stateParams.userId).then(winCb, failCb);
+    championsForUser: function ($stateParams, cdUsersService, auth) {
```

Comment:
I 'd prefer to decuplate the resolve as they don't serve the same data :  it was "get the champions of the visible profile" and now it's also "get the champion of the connected user"?